app/controllers/registrations_controller.rb

```diff
@@ -6,7 +6,9 @@ def new
   
   def create
     @user = User.new(user_params)
+    @cart = ShoppingCart.new
       if @user.save
+        @user.shopping_cart = @cart
```

Comment:
Use a callback for creating cart. What if I create a user from rails console?

Reply:
Done